.github/workflows/release.yml

```diff
@@ -7,17 +7,31 @@ on:
   workflow_dispatch:
 
 jobs:
+  version:
+    outputs:
+      version: ${{ steps.calculate_version.outputs.version }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - id: calculate_version
+      run: |
+        git fetch --tags
```

Comment:
The command `git fetch --tags` is already done inside `next_version.sh`.

Reply:
Good catch, thank you!